client/views/admin/emailInbox/EmailInboxForm.js

```diff
@@ -204,7 +205,17 @@ export default function EmailInboxForm({ id, data }) {
 		}
 	});
 
-	const canSave = hasUnsavedChanges && name && (email && isEmail(email))
+
+	const checkEmailExists = useMutableCallback(async () => {
+		if (isEmail(email)) {
+			const result = await emailAlreadyExistsAction();
```

Comment:
After creating a proper endpoint for searching emails, you will get the results here(array).
Then, to be an invalid email you'll loop the array([array find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)) and look for the same email belonging to another record.